### PR TITLE
Move source files in dedicated folder

### PR DESCRIPTION
**Pull request by umbynos**
The objective of this pr is to:
- [X] move source files in a dedicated `src` folder
- [X] refactor `Dockerfile` accordingly

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,9 @@
-FROM python:3
+FROM python:3-slim
```
Comment (umbynos) [inline Dockerfile:1]: I've also tried to optimize the size of the docker image. This change reduces the image size form almost 1GB to 358MB

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 
-COPY bot/ ./bot/
-COPY game/ ./game/
-COPY res/ ./res/ 
-COPY util/ ./util/
-COPY main.py requirements.txt ./
-RUN pip install -r requirements.txt
+RUN apt-get update && apt-get -y install gcc
```
Comment (umbynos) [inline Dockerfile:3]: This was mandatory because `python:3-slim` does not come with build tools and some dependencies need to be compiled.

**Comment by fily1212** [comment]
In Italian we say "gasa", referring to python slim and the lighter image

**Review by fily1212 — APPROVED** [review]
Top!!!

**Pull request merged**